Field 'browser' doesn't contain a valid alias configuration

Had trouble using the default steps, sorry if I was expected to configure something else as well.

## Installing (all good)

```zsh
~/Projects> git clone https://github.com/superfly/onehostname.git
Cloning into 'onehostname'...
remote: Counting objects: 68, done.
remote: Compressing objects: 100% (41/41), done.
remote: Total 68 (delta 25), reused 59 (delta 20), pack-reused 0
Unpacking objects: 100% (68/68), done.
~/Projects> cd onehostname/
```

## Running (problems)

```zsh
~/P/onehostname master> fly server
Using /Users/MyUser/Projects/onehostname as working directory.
Generating Webpack config...
Compiling app w/ options: { watch: true, uglify: false }
Server listening on :::3000
2018-04-25T03:35:33.881Z - info: Config watch (add: .fly.yml)
Error: Hash: 5f93e961599d66addf352d47e686f5cee2fa1774
Version: webpack 3.11.0
Time: 168ms
          Asset     Size  Chunks             Chunk Names
      bundle.js  4.84 kB       0  [emitted]  main
bundle.map.json  4.89 kB       0  [emitted]  main
   [0] ./index.js 1.84 kB {0} [built]

ERROR in ./index.js
Module not found: Error: Can't resolve './lib/backends' in '/Users/MyUser/Projects/onehostname'
resolve './lib/backends' in '/Users/MyUser/Projects/onehostname'
  using description file: /Users/MyUser/Projects/onehostname/package.json (relative path: .)
    Field 'browser' doesn't contain a valid alias configuration
  after using description file: /Users/MyUser/Projects/onehostname/package.json (relative path: .)
    using description file: /Users/MyUser/Projects/onehostname/package.json (relative path: ./lib/backends)
      no extension
        Field 'browser' doesn't contain a valid alias configuration
        /Users/MyUser/Projects/onehostname/lib/backends doesn't exist
      .js
        Field 'browser' doesn't contain a valid alias configuration
        /Users/MyUser/Projects/onehostname/lib/backends.js doesn't exist
      as directory
        /Users/MyUser/Projects/onehostname/lib/backends doesn't exist
[/Users/MyUser/Projects/onehostname/lib/backends]
[/Users/MyUser/Projects/onehostname/lib/backends.js]
[/Users/MyUser/Projects/onehostname/lib/backends]
 @ ./index.js 1:0-37

ERROR in ./index.js
Module not found: Error: Can't resolve './lib/utilities' in '/Users/MyUser/Projects/onehostname'
resolve './lib/utilities' in '/Users/MyUser/Projects/onehostname'
  using description file: /Users/MyUser/Projects/onehostname/package.json (relative path: .)
    Field 'browser' doesn't contain a valid alias configuration
  after using description file: /Users/MyUser/Projects/onehostname/package.json (relative path: .)
    using description file: /Users/MyUser/Projects/onehostname/package.json (relative path: ./lib/utilities)
      no extension
        Field 'browser' doesn't contain a valid alias configuration
        /Users/MyUser/Projects/onehostname/lib/utilities doesn't exist
      .js
        Field 'browser' doesn't contain a valid alias configuration
        /Users/MyUser/Projects/onehostname/lib/utilities.js doesn't exist
      as directory
        /Users/MyUser/Projects/onehostname/lib/utilities doesn't exist
[/Users/MyUser/Projects/onehostname/lib/utilities]
[/Users/MyUser/Projects/onehostname/lib/utilities.js]
[/Users/MyUser/Projects/onehostname/lib/utilities]
 @ ./index.js 2:0-42
    at Watching.handler (/Users/MyUser/.nvm/versions/v8.9.1/lib/node_modules/@fly/fly/src/utils/build.ts:41:16)
    at Watching._done (/Users/MyUser/.nvm/versions/v8.9.1/lib/node_modules/@fly/fly/node_modules/webpack/lib/Compiler.js:106:8)
    at compiler.emitRecords.err (/Users/MyUser/.nvm/versions/v8.9.1/lib/node_modules/@fly/fly/node_modules/webpack/lib/Compiler.js:79:19)
    at Compiler.emitRecords (/Users/MyUser/.nvm/versions/v8.9.1/lib/node_modules/@fly/fly/node_modules/webpack/lib/Compiler.js:371:38)
    at compiler.emitAssets.err (/Users/MyUser/.nvm/versions/v8.9.1/lib/node_modules/@fly/fly/node_modules/webpack/lib/Compiler.js:62:20)
    at applyPluginsAsyncSeries1.err (/Users/MyUser/.nvm/versions/v8.9.1/lib/node_modules/@fly/fly/node_modules/webpack/lib/Compiler.js:364:12)
    at next (/Users/MyUser/.nvm/versions/v8.9.1/lib/node_modules/@fly/fly/node_modules/tapable/lib/Tapable.js:218:11)
    at Compiler.compiler.plugin (/Users/MyUser/.nvm/versions/v8.9.1/lib/node_modules/@fly/fly/node_modules/webpack/lib/performance/SizeLimitsPlugin.js:99:4)
    at Compiler.applyPluginsAsyncSeries1 (/Users/MyUser/.nvm/versions/v8.9.1/lib/node_modules/@fly/fly/node_modules/tapable/lib/Tapable.js:222:13)
    at Compiler.afterEmit (/Users/MyUser/.nvm/versions/v8.9.1/lib/node_modules/@fly/fly/node_modules/webpack/lib/Compiler.js:361:9)
    at require.forEach.err (/Users/MyUser/.nvm/versions/v8.9.1/lib/node_modules/@fly/fly/node_modules/webpack/lib/Compiler.js:350:15)
    at /Users/MyUser/.nvm/versions/v8.9.1/lib/node_modules/@fly/fly/node_modules/async/dist/async.js:473:16
    at iteratorCallback (/Users/MyUser/.nvm/versions/v8.9.1/lib/node_modules/@fly/fly/node_modules/async/dist/async.js:1050:13)
    at /Users/MyUser/.nvm/versions/v8.9.1/lib/node_modules/@fly/fly/node_modules/async/dist/async.js:958:16
    at /Users/MyUser/.nvm/versions/v8.9.1/lib/node_modules/@fly/fly/node_modules/graceful-fs/graceful-fs.js:43:10
    at FSReqWrap.oncomplete (fs.js:135:15)
## had to ctrl+c here
 ~/P/onehostname master> npm install
## tried again after npm install and npm install -g typescript
```

Node is 8.9.1

--- a/README.md
+++ b/README.md
@@ -8,7 +8,7 @@ Backend types are defined in [src/backends.js](src/backends.js). These create a 
 
 ## Try it out
 
-Install fly, clone this repo, compile typescrpt, start the server.
+Install fly, clone this repo, compile typescript, start the server.
 
 ```bash
 $ npm install -g @fly/fly typescript

--- a/README.md
+++ b/README.md
@@ -8,12 +8,13 @@ Backend types are defined in [src/backends.js](src/backends.js). These create a 
 
 ## Try it out
 
-Install fly, clone this repo, start the server.
+Install fly, clone this repo, compile typescrpt, start the server.
 
 ```bash
-$ npm install -g @fly/fly
+$ npm install -g @fly/fly typescript
 $ git clone https://github.com/superfly/onehostname.git
 $ cd onehostname
+$ tsc
 $ fly server
 ```
 

--- a/README.md
+++ b/README.md
@@ -11,10 +11,10 @@ Backend types are defined in [src/backends.js](src/backends.js). These create a 
 Install fly, clone this repo, compile typescript, start the server.
 
 ```bash
-$ npm install -g @fly/fly typescript
 $ git clone https://github.com/superfly/onehostname.git
 $ cd onehostname
-$ tsc
+$ npm install
+$ npx tsc
 $ fly server
 ```
 

